content/en/blog/_posts/2021-11-18-honor-pv-reclaim-policy.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: blog
+title: "Kubernetes 1.23 Feature: Honor PV Reclaim Policy"
+date: 2021-11-18T10:00:00-08:00
+slug: kubernetes-1-23-feature-honor-pv-reclaim-policy
+---
+
+**Authors:** Deepak Kinni, VMware
+
+[Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) are associated with [Reclaim Policy](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaim-policy). The Reclaim Policy is used to determine the actions that need to be taken by the storage backend on deletion of the PV. In case where the Reclaim Policy is `Delete` the expectation is that the storage backend releases the storage resource that was allocated for the PV. In essence, the Reclaim policy needs to honored on PV deletion.
+
+## What is the existing behavior?
+
+Normally, if the volume is to be deleted, then the expectation is to delete the PVC for a bound PV-PVC pair. However, there is no restriction to delete a PV prior to deleting a PVC.
```

Comment:
We are using “PVC” here without first introducing the term. Some readers will be puzzled, and we should help them with a (brief) summary of how PVCs and PVs are related, plus a hyperlink for further reading.